pyCGM_Single/Pipelines.py

```diff
@@ -53,9 +53,9 @@ def butterFilter(data, cutoff, Fs):
     First, we create a sin wave and add noise to it.
 
     >>> from numpy import arange, around, pi, random, shape, sin
-    >>> Fs = 360
+    >>> Fs = 360.0
     >>> t = 1
-    >>> x = arange(0,t,1/Fs)
+    >>> x = arange(0,2,1/Fs)
```

Comment:
Why did you change this?

Reply:
The 360.0 was intentional to fix a division by 0 error that was only happening on the Python 2 build, the t -> 2 is an error and will be removed.